whois-commons/src/main/resources/patch/whois-1.106.sql

```diff
@@ -0,0 +1,7 @@
+-- delete default maintainer history tables. Not longer in use.
+
+DROP TABLE IF EXISTS `default_maintainer_history`;
+DROP TABLE IF EXISTS `default_maintainer_sync_history`;
+
```

Comment:
It should be Internals patch file?

Reply:
check internals-1.102.sql as an example

Reply:
what is the difference here? how could I know that this is an internals?

Reply:
By which schema table exists in.  If tables exists in whois schema it should be in whois patch file, if table exists in internals schema it should be in internals schema

Reply:
ok, makes sense